### PR TITLE
[sram/dv] Sign off V2S

### PR DESCRIPTION
**Pull request by weicaiyang**
Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

**Comment by weicaiyang** [comment]
All the SRAM_CTRL related AIs in #11039 are done. 
There are 2 other items to update the checklist, which need the coming RFC this Friday for approval. I don't think we need to wait for that before signing off this block.

File: hw/ip/sram_ctrl/doc/checklist.md [hunk 1]
```diff
@@ -222,10 +222,10 @@ Review        | [V3_CHECKLIST_SCOPED][]                 | Done        |
 
  Type         | Item                                    | Resolution  | Note/Collaterals
 --------------|-----------------------------------------|-------------|------------------
-Documentation | [SEC_CM_TESTPLAN_COMPLETED][]           | Not Started |
-Tests         | [FPV_SEC_CM_VERIFIED][]                 | Not Started |
-Tests         | [SIM_SEC_CM_VERIFIED][]                 | Not Started |
-Review        | [SEC_CM_DV_REVIEWED][]                  | Not Started |
+Documentation | [SEC_CM_TESTPLAN_COMPLETED][]           | Done        |
+Tests         | [FPV_SEC_CM_VERIFIED][]                 | Done        | Sub-module prim_count is proven in FPV
```
Comment (sriyerg) [inline hw/ip/sram_ctrl/doc/checklist.md:226]: @cindychip 
We are also proving the IP itself also in an FPV testbench, right? To prove the error from `prim_counts` always reaches the alert output unhindered.

**Review by sriyerg — COMMENTED** [review]
Didn't we add a few more V2S items? I suggest making a PR updating the checklists first.

**Comment by weicaiyang** [comment]
> Didn't we add a few more V2S items? I suggest making a PR updating the checklists first.

yes, I made a RFC #11090 for it. Need to go over it in the TC meeting this Friday and then update the final checklist at #11053
I feel this block may not need to wait for the checklist update, but I'm fine to wait until the checklist is landed.

File: hw/ip/sram_ctrl/doc/checklist.md (hunk 1 above)
Comment (weicaiyang) [reply hw/ip/sram_ctrl/doc/checklist.md:226]: yes, Cindy helped to add it and it's passing now according to https://reports.opentitan.org/hw/top_earlgrey/formal/summary.html.

**Review by matutem — APPROVED** [review]
LGTM

**Review by cindychip — APPROVED** [review]

**Review by msfschaffner — APPROVED** [review]
It is probably ok to move on.
If we have to update the checklist, we can do that later on as well.

**Pull request merged**